src/feature/gaussian.hh

```diff
@@ -9,101 +9,116 @@
 #include "lib/utils.hh"
 #include "lib/timer.hh"
 #include "common/common.hh"
-namespace pano {
+namespace pano
+{
```

Comment:
most of this PR are not related to compiler warnings

Reply:
Have a good day, automatic re-indentation process was accomplished by my IDE, I am disabling it in order to change the correct lines only.

Reply:
Comments added on really helpful changes.